app/com/linkedin/drelephant/tuning/obt/ParameterGenerateManagerOBTAlgoPSOIPSOImpl.java

```diff
@@ -86,23 +61,16 @@ private void setDefaultParameterValues(TuningAlgorithm tuningAlgorithm, JobDefin
           tuningParameterConstraint.constraintType = TuningParameterConstraint.ConstraintType.BOUNDARY;
           tuningParameterConstraint.save();
         }
-      }
-      else{
+      } else {
         logger.debug(" Parameter constraints already added . Hence not adding them");
       }
     } catch (Exception e) {
       logger.debug(
-          " Error in setting up intial parameter constraint . IPSO will not work in this case ." + e.getMessage(),e);
+          " Error in setting up intial parameter constraint . IPSO will not work in this case ." + e.getMessage(), e);
```

Comment:
Guard with isDebugEnabled and once we have passed exception as a parameter, I dont think e.getMessage is required.